server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -22,11 +28,31 @@ export class MediaRepository implements IMediaRepository {
         return;
 
       case 'jpeg':
-        await sharp(input, { failOnError: false })
-          .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-          .jpeg()
-          .rotate()
-          .toFile(output);
+        if (!options.raw) {
+          await sharp(input, { failOnError: false })
+            .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
+            .jpeg()
+            .rotate()
+            .toFile(output);
+        } else {
+          try {
+            await stat(output);
+            await unlink(output);
+          } catch {}
+          const command = `darktable-cli '${input}' '${output}' --apply-custom-presets false --width ${options.size} --height ${options.size} --core --library :memory:`;
```

Comment:
I've noticed that `--width/--height` options of `darktable-cli` limit the _maximum_ width/height of the image. But we need to guarantee that thumbnail width/height is as small as possible but _not less_ than `options.size`